portal/src/graphql/adminapi/UserDetailSummary.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import cn from "classnames";
+import { Context, FormattedMessage } from "@oursky/react-messageformat";
+
+import proPicPlaceholder from "../../image/profile-pic-placeholder.svg";
+
+import { formatDatetime } from "../../util/formatDatetime";
+import { UserInfo } from "../../util/user";
+
+import styles from "./UserDetailSummary.module.scss";
+
+interface UserDetailSummaryProps {
+  className?: string;
+  userInfo: UserInfo;
+  createdAtISO: string | null;
+  lastLoginAtISO: string | null;
+}
+
+const UserDetailSummary: React.FC<UserDetailSummaryProps> = function UserDetailSummary(
+  props: UserDetailSummaryProps
+) {
+  const { userInfo, createdAtISO, lastLoginAtISO, className } = props;
+  const { username, email } = userInfo;
+  const { locale } = React.useContext(Context);
+
+  const formatedSignedUp = React.useMemo(() => {
+    return formatDatetime(locale, createdAtISO);
+  }, [locale, createdAtISO]);
+  const formatedLastLogin = React.useMemo(() => {
+    return formatDatetime(locale, lastLoginAtISO);
+  }, [locale, lastLoginAtISO]);
+
+  return (
+    <div className={cn(styles.root, className)}>
+      <img src={proPicPlaceholder} className={styles.profilePic} />
```

Comment:
Use [Persona](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona) for placeholder. Remove the dummy image.